.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
You should also ignore all of your .DS_Store files, since those are showing up here and not necessary.